ocp_resources/virtual_machine_instance.py

```diff
@@ -204,11 +205,48 @@ def get_xml(self):
         Returns:
             xml_output(string): VMI XML in the multi-line string
         """
+        cmd = f"virsh {self.virt_launcher_pod_hypervisor_connection_uri} dumpxml {self.namespace}_{self.name}"
```

Comment:
`f"virsh {self.virt_launcher_pod_hypervisor_connection_uri} dumpxml {self.namespace}_{self.name}"`

Please create a function _virsh_cmd that returns the cmd.
it this case it will get `dumpxml`

Reply:
Done